cmake/ProjectPaillier.cmake

```diff
@@ -5,26 +5,24 @@ if (CRYPTO_EXTENSION)
      PREFIX ${CMAKE_SOURCE_DIR}/deps
         DOWNLOAD_NO_PROGRESS 1
         GIT_REPOSITORY https://github.com/FISCO-BCOS/paillier.git
-	    GIT_TAG a2a0e11eb9f4c6eb2d5b7bc3de36eeceda44529a
+	    GIT_TAG 10ebe553d6d238e6b88fbf722509709db792dcb4
         BUILD_IN_SOURCE 1
-        SOURCE_SUBDIR ./paillier_cpp/
+        # SOURCE_SUBDIR ./paillier_cpp/
```

Comment:
add a comment to explain or delete it. 
please add this compile option to TravisCI